hubspot/serializers.py

```diff
@@ -41,57 +44,80 @@ def get_purchaser(self, instance):
 
     def get_price(self, instance):
         """Get a string of the price"""
-        return instance.price.to_eng_string()
+        return instance.bootcamp_run.personal_price(instance.user).to_eng_string()
 
     def get_bootcamp_name(self, instance):
         """Get the name of the bootcamp"""
         return instance.bootcamp_run.bootcamp.title
 
+    def get_application_stage(self, instance):
+        """Get the application stage"""
+        state = instance.state
+        if state == AppStates.AWAITING_USER_SUBMISSIONS.value:
+            submission_subquery = instance.submissions.all()
```

Comment:
Can you turn all of this into an API function and call it from here? Something like `get_required_submission_type(application)`